cdap-ui/app/cdap/styles/common.scss

```diff
@@ -198,4 +198,13 @@ body {
     overflow: hidden;
     text-overflow: ellipsis;
   }
+
+  /* Typography */
+  .caption-1 {
```

Comment:
nit: Since we are adding this in react side can this come in the theme we define from material-ui? I understand we don't use the material-ui here yet but we have a context that is added at the top level. Checking if it will be a huge effort to change this to material-ui component.

If not this will be a duplication we have to get away with when we transition to material-ui